src/util/image/MapCreator.java

```diff
@@ -544,7 +554,20 @@ private static String getValue(final String arg) {
 
   private static void handleCommandLineArgs(final String[] args) {
     final String[] properties = getProperties();
-    if (args.length == 1) {
+    boolean laf = false;
```

Comment:
Really pls spell out variable names: `laf`
I *did* not know what this stood for, had to search through the code to figure it out.
Also, it does not appear this variable is being used.